src/en/charms-storage-ceph.md

```diff
@@ -0,0 +1,123 @@
+Title: Installing Ceph
+
+# Installing Ceph
+
+Here we present one way to install a rudimentary Ceph cluster with Juju backed
+by an AWS cloud. See [Ceph documentation][upstream-ceph-documentation] to find
+out more about Ceph.
+
+Below, we install three Ceph monitors and three OSD systems. Each OSD system
+has two 32 GiB OSDs and one 8 GiB volume dedicated to the OSD journal. All
+volumes are EBS-based and magnetic (i.e. not SSD).
+
+```bash
+juju deploy -n 3 ceph-mon
+juju deploy -n 3 ceph-osd --storage osd-devices=ebs,32G,2 --storage osd-journals=ebs,8G,1
+juju add-relation ceph-mon ceph-osd
+```
+
+This will take a while. Presented below is the final output to the `juju status`
+command:
+
+```no-highlight
+Model  Controller      Cloud/Region   Version       SLA
+ceph   aws-controller  aws/us-east-1  2.3-alpha1.1  unsupported
```

Comment:
I'd go ahead and trim the "alpha1.1 off the version just so that this stays up to date a bit more.